### ISSUE TITLE
Doesn't extract anything, or it's encoded wrong

### ISSUE THREAD

**Issue by vegardsando** [cadon/ARKStatsExtractor#496]
Either I'm doing something terribly wrong, or something's off. When trying to read the stat of any dino, it doesn't read anything, or it's incorrect. Like my tribe name is read as "!-!>Y|>!-_+PY" which is.. what. And the owner is read as ",_,,_,".

**Comment by cadon** [comment]
Hi, for letters the recognition sometimes is really bad. Did you make sure you use the correct resolution and your UI is not scaled to another value than 100%?

**Comment by vegardsando** [comment]
Ah there you go, it was the scaling. That makes sense, it read values from specific coordinates on the screen. Thank you

**Comment by cadon** [comment]
Yes. You can create an OCR-config-file just for the scaling you're using, it would require adjusting the reading-label-positions and maybe also the font-sizes. Have a look here on how to do it if you're interested: https://github.com/cadon/ARKStatsExtractor/wiki/OCR

**Comment by DJQuadRocks** [comment]
I'm trying to make a custom OCR but it's not grabbing the stats after it focuses the ARK window. 

My screen resolution is 1980x1080 and my ARK window is 1600x900 windowed, if that helps.

![image](https://user-images.githubusercontent.com/6164861/31320263-c77baccc-ac3f-11e7-952d-4678975c8e98.png)

**Comment by cadon** [comment]
Hi @DJQuadRocks, what resolution does the screenshot have, 1600x900 or 1920x1080? If you use the "Read Values"-option, the resolution will be the one of windows (probably 1920x1080 for you). Also, the OCR-config for 1920x1080 is already included in with the application, that should work more or less.

**Comment by DJQuadRocks** [comment]
The screenshot is what my ARK resolution is - 1600x900. It also was taken after I moved the positions of things like Health, Stamina, etc.

Are you familiar with AutoHotKey? It has some OCR libraries that may help. See https://github.com/camerb/AHKs/blob/master/camerb-OCR-function/OCR.ahk and https://autohotkey.com/board/topic/69127-ocrahk-library-for-recognizing-text-in-images.

**Comment by cadon** [comment]
Hi, the resolution has to be the resolution of windows, i.e. 1920x1080 in your case. If you used the steam-screenshot-function, the resulting screenshots will be in the resolution of the game (i.e. 1600x900 for you). Smart Breeding captures the screenshot in the windows-resolution and has to be calibrated to that. This means, you can simply use the provided OCR-config for 1920x1080.

The reason the windows-resolution is taken is based on the way C# captures screenshots, I haven't found a way to take screenshots in the ingame-resolution yet.

Thanks for the links, I'll have a look at them.
